src/scss/components/checkbox/_index.scss

```diff
@@ -1,6 +1,7 @@
-@import (reference) 'vars';
-@import (reference) 'mixins';
-@import (reference) 'components/form/index';
+@import "vars";
+@import "mixins";
```

Comment:
Move to `_config.scss`

Reply:
Moved to `src/scss/components/checkbox/_config.scss`:
```css
@import "mixins";
```